### ISSUE TITLE
Broken history formation of files in Nextcloud group folders

### ISSUE THREAD

**Issue by natoponen** [ONLYOFFICE/onlyoffice-nextcloud#671]
Hello!

I am experiencing an issue when working with history of files in Nextcloud group folders. There are no revisions and history formation seems to be broken. Assume I make 4 versions of document (either docx, xlsx or pptx) then open version history and here is what I see:

- ver.4 is the latest version
- ver.3 is what should be ver.1
- ver.2 is as it should be
- ver.1 is what should be ver.3

In other words it forms this way: latest version is what it should be and then all other versions in reverse order.

My environment:
- OnlyOffice version: 7.0.1.37
- Nextcloud version: 24.0.1
- onlyoffice-nextcloud connector version: 7.5.4

Is there any way to fix it? Thanks in advance!
